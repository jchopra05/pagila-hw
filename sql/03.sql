/*
 * Find all last names of actors whose last names contain the letters LI (case insensitive).
 * Order results alphabetically.
 */

SELECT DISTINCT last_name
FROM public.actor
WHERE LOWER(last_name) LIKE '%li%'
ORDER BY last_name ASC;
