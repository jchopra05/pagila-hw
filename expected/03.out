 last_name 
-----------
 CHAPLIN
 JOLIE
 OLIVIER
 WILLIAMS
 WILLIS
(5 rows)

